Testing/igtlioFixture.cxx

```diff
@@ -82,7 +83,7 @@ bool igtlioClientServerFixture::ConnectClientToServer()
 bool igtlioClientServerFixture::LoopUntilEventDetected(igtlioLogicFixture* logic, int eventId, int count)
 {
 
-  double timeout = 2000;
+  double timeout = 2;
```

Comment:
if the units have changed, could you suffix the variable name with units? `double timeoutSec(2);`